src/probot.ts

```diff
@@ -34,7 +32,7 @@ export class Probot {
   }
 
   public webhooks: ProbotWebhooks;
-  public log: DeprecatedLogger;
```

Comment:
Yeah, took this from probot-ng along with capture-log-output. Is there a more elegant way?

Reply:
It seems fine

Reply:
then i mark this convo as resolved ;)